web/docs/internal/gestures/index.md

```diff
@@ -0,0 +1,41 @@
+# Gesture Recognizer design - index
+
+GestureRecognizer is an engine designed to interpret mouse event paths and touch event paths into sequences of externally-specified [gesture components](./glossary.md).  This is accomplished via definition of gesture-component "models", which the engine is then able to interpret when "processing" the input.  Each gesture-component model may be defined from its own perspective independently of any other, though effort may be required to identify and carefully define components that are shared by two or more gesture types.
+
+There are a few different aspects to GestureRecognizer, each with its own internal doc.
+
+- [Gesture modeling](./gesture-modeling.md) refers to the specification of gestures to be interpreted from the user's input.
+- [Gesture processing](./gesture-processing.md) refers to the process of interpreting the user's input as gestures.
+- The [Glossary](./glossary.md) provides definitions for some of the terms seen among the internal documentation pages.
+
+There is also documentation regarding some of the trickier aspects of [configuring the region that accepts gestures](./recognizer-configuration.md)
+
+Keyman configures two separate instances of this gesture engine - one for main keyboard body and another for the predictive-text banner.  In both cases, gestures (from the user perspective) are broken own into components, each corresponding to different movement patterns that comprise a each gesture type.  Once components are recognized, they are added into a sequence that represents the causal relationship among the different components of the gesture and ensures that it may continue if appropriate for that type of gesture.
```

Comment:
```suggestion
Keyman configures two separate instances of this gesture engine - one for main keyboard body and another for the predictive-text banner.  In both cases, gestures (from the user perspective) are broken down into components, each corresponding to different movement patterns that each comprise a gesture type.  Once components are recognized, they are added into a sequence that represents the causal relationship among the different components of the gesture and ensures that it may continue if appropriate for that type of gesture.
```
(I think "each" was in the wrong place, but I'm not entirely sure where it should be...)